rviz_default_plugins/package.xml

```diff
@@ -33,6 +33,9 @@
   <depend>resource_retriever</depend>
   <depend>rviz_common</depend>
   <depend>rviz_rendering</depend>
+  <depend>tf2</depend>
+  <depend>tf2_ros</depend>
+  <depend>tf2_geometry_msgs</depend>
```

Comment:
nitpick: Alphabetical ordering (maybe in a few other places too)

Reply:
Fixed df50f8c